soc/Kconfig.v2

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+
+# SPDX-License-Identifier: Apache-2.0
+
+# Load all SoCs.
+# SoCs that has transistioned to SoC scheme v2 are promptless, and thus
+# unselectable by the user or thorugh configuration files.
+#
+# SoC that are still using scheme v1 are still having a prompt but is loaded as
+# they offer board maintainers the possibility to use board scheme v2.
+
+config SOC
+	string
+	help
+	  SoC name which can be found under soc/<arch>/<soc name>.
```

Comment:
This v2 file still does not seem to address the use case in https://github.com/zephyrproject-rtos/zephyr/issues/51831.

Reply:
good point.
That should definitely change.

Reply:
Has this been addressed?

Reply:
Rephrased